styles/tailwind.config.js

```diff
@@ -29,13 +29,17 @@ module.exports = {
         wiggleInverse: 'wiggle 5s ease-in-out alternate-reverse infinite',
         push: 'push 6s ease-in-out alternate infinite',
         pushAlt: 'push 4s ease-in-out alternate infinite',
-        pushInverse: 'push 5s ease-in-out alternate-reverse infinite'
+        pushInverse: 'push 5s ease-in-out alternate-reverse infinite',
       },
     },
     fontFamily: {
-      'sans': ['Inter, Overpass, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"'],
-      'serif': ['Piazzolla, ui-serif, Georgia, Cambria, "Times New Roman", Times, serif'],
-      'mono': ['Menlo, ui-monospace, SFMono-Regular, Monaco, Consolas, "Ubuntu Mono", "Liberation Mono", "Courier New", monospace']
+      sans: [
```

Comment:
Tailwind docs technically do not support this, but it appears to work though

https://tailwindcss.com/docs/font-family